linz-data-importer/Makefile

```diff
@@ -74,7 +74,7 @@ default: compile
 compile: $(COMPILED_RESOURCE_FILES)
 
 %.py : %.qrc $(RESOURCES_SRC)
-	pyrcc4 -o $*.py  $<
+	pyrcc5 -o $*.py  $<
```

Comment:
Shouldn't this tool be mentioned in some dependency file or documentation? I can't find it anywhere else.

Reply:
It get's installed with pyqt5 I believe which is in the requirements list. https://github.com/linz/linz-data-importer/blob/e23152e20e80dbf62aad3157567c17ef8fe80b2e/requirements-dev.txt#L22

But seems you can't refer to it as pyrcc. It has to be pyrcc{4,5}.